src/mlpack/core/data/string_cleaning.hpp

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file string_cleaning.hpp
+ * @author jeffin sam
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_DATA_STRING_CLEANING_HPP
+#define MLPACK_CORE_DATA_STRING_CLEANING_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <boost/range/algorithm/remove_if.hpp>
+#include <boost/algorithm/string/classification.hpp>
+#include "mlpack/core/boost_backport/boost_backport_string_view.hpp"
+#include <mlpack/core/data/tokenizer/strtok.hpp>
+#include <boost/functional/hash.hpp>
+
+std::string stopwords =
+#include "mlpack/core/data/stopwords.txt"
+;
+
```

Comment:
Linter failes here, `;`, I have done this to include the file.

Reply:
Are you playing to keep this in the final version? I thought it might be useful to allow the user to pass a stop word file?

Reply:
I removed the file, since both of you suggested to have an api, where user passes the file.